packages/web/components/Landing/LandingConnectButton.tsx

```diff
@@ -0,0 +1,176 @@
+/* eslint-disable no-nested-ternary */
+import {
+  Box,
+  Button,
+  Icon,
+  Spinner,
+  Text,
+  Tooltip,
+  useBreakpointValue,
+  useToast,
+  VStack,
+} from '@metafam/ds';
+import { MetaLink } from 'components/Link';
+import { useWeb3 } from 'lib/hooks';
+import { useEffect, useState } from 'react';
+import { GoSignIn, GoSignOut } from 'react-icons/go';
+import { shortenAddress } from 'utils/stringHelpers';
+
+export const LandingConnectButton = ({ isIconStyle = false, ...props }) => {
+  const {
+    connect,
+    disconnect,
+    connected,
+    connecting,
+    address,
+    chainId,
+    provider,
+  } = useWeb3();
+  const [wrongNetwork, setWrongNetwork] = useState(false);
+  const spinnerSize = useBreakpointValue({ base: 'sm', '2xl': 'md' });
+  const toast = useToast();
+
+  useEffect(() => {
+    if (connected && address && provider) {
+      const networkName = provider.network.name;
+      const shortAddress = shortenAddress(address);
+      console.log('Connected to', networkName, 'with address', shortAddress);
```

Comment:
Also we may want to remove all these console logs. If you want them during testing we can wrap them all into a helper function which logs based on an env variable.

Reply:
> Also we may want to remove all these console logs. If you want them during testing we can wrap them all into a helper function which logs based on an env variable.

they're definitely helping during testing so yeah, a helper function is a good idea. 👍🏻

Reply:
✔️